packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -24,6 +32,21 @@ import {useThemebundling} from '@shopify/cli-kit/node/context/local'
 import {fileExists, touchFile, writeFile} from '@shopify/cli-kit/node/fs'
 import {getPathValue} from '@shopify/cli-kit/common/object'
 
+export const CONFIG_EXTENSION_IDS = [
+  AppAccessSpecIdentifier,
+  AppHomeSpecIdentifier,
+  AppProxySpecIdentifier,
+  BrandingSpecIdentifier,
+  PosSpecIdentifier,
+  appPrivacyComplienceSpec,
+  WebhookSubscriptionSpecIdentifier,
+  WebhooksSpecIdentifier,
+  '0f844fe3-fee9-45ae-9b68-7e596b3eaaa9',
+  '10457bd6-aeea-4f92-ab90-a9ab1e471276',
+  '0ea86845-466c-4f0e-b7f1-9d9496d93f4a',
+  '406bb2df-eaa4-44bc-860d-c714f4b65def',
```

Comment:
@amcaplan do you remember what are these ones?

Reply:
Maybe they were the UUIDs for the Big Four modules? The number 4 generally suggests that direction...

Reply:
[This commit](https://github.com/Shopify/cli/commit/565ea41e7342755d487409228e93ed8352c8a1d0) certainly indicates as such!

Reply:
It's a bit hard to tell from the context of this PR whether we're talking about the new app management API, the existing Partners API, or both. The new app management API uses the `static` UID strategy from the specification for what we used to call config modules, so technically there's no need to specify a `uid` for them at all -- it will be inferred server-side to be the specification identifier.